### DISCUSSION TITLE
A few ideas

### DISCUSSION THREAD

**Discussion by kais-grati** [flxzt/rnote#1240] [Ideas]
- Hold pen to auto make shapes
- Dashed borders for shapes
- Highlighter pen
- Auto save on change
- Strikethrough feature where you scribble on text to delete it

**Comment by Doublonmousse** [comment]
> * Hold pen to auto make shapes
-> #384 and #1175 for a draft PR
> * Dashed borders for shapes
-> #1210 (in progress) and #852 and #375 for the issues
> * Highlighter pen
#891 and #449
> * Auto save on change
This one doesn't have an opened issue.
I think it wouldn't work well if saving uses the current file format (saving the full state on each pen up event then compressing it) as it will probably be too slow and compute heavy. Maybe changing temporarily the format to be uncompressed whilst doing that then re compressing it when quitting would make it easier to do. Though having a diff system instead of a full save would be faster.
> * Strikethrough feature where you scribble on text to delete it
#789 ? Maybe scratch is a little bit better than strikethrough ? This has to be optional and having the "math fractions erase everyting bug" wouldn't be great
